src/SFA.DAS.ApprenticeCommitments.Web.AcceptanceTests/Bindings/Web.cs

```diff
@@ -1,18 +1,18 @@
-﻿using System.Collections.Generic;
-using System.Net.Http;
 using Microsoft.AspNetCore.Mvc;
 using SFA.DAS.ApprenticeCommitments.Web.AcceptanceTests.Hooks;
 using SFA.DAS.ApprenticeCommitments.Web.Startup;
+using System.Collections.Generic;
+using System.Net.Http;
 using TechTalk.SpecFlow;
 
 namespace SFA.DAS.ApprenticeCommitments.Web.AcceptanceTests.Bindings
 {
     [Binding]
     public class Web
     {
-        public static HttpClient Client { get; set; }
-        public static LocalWebApplicationFactory<ApplicationStartup> Factory { get; set; }
-        public Hook<IActionResult>  ActionResultHook;
+        public HttpClient Client { get; set; }
+        public LocalWebApplicationFactory<ApplicationStartup> Factory { get; set; }
+        public Hook<IActionResult> ActionResultHook;
```

Comment:
> Also, in the original implementation these 2 scenarios they cannot live in the same feature file, but they can with this implementation

That's exactly the reason for this change :)  I tried to add the second scenario and it was failing because the `LastPageResult` hook was null (due to the `TestContext` being recreated anew for each scenario)

Let's discuss the approach regarding scenarios